crates/dscvr-telemetry-util/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+pub mod axum {
+    use axum::{extract::MatchedPath, middleware::Next, response::Response, routing::get, Router};
+    use http::Request;
+    use metrics_exporter_prometheus::{BuildError, PrometheusBuilder};
+    use std::time::Instant;
+
+    // Takes an existing axum router, installs the prometheus metrics recorder and
+    // injects the metrics endpoint into the router after the handler layer is installed so that
+    // `/metrics` route itself is not included in the routing layer metrics measured
+    pub fn install_axum_metrics(app: Router) -> Result<Router, BuildError> {
+        let handle = PrometheusBuilder::new().install_recorder()?;
```

Comment:
Should we pass the handle in as an argument? How are we going to configure the recorder with this approach?